main.go

```diff
@@ -56,76 +56,128 @@ var (
 	verbose          bool
 )
 
+func init() {
+	flag.StringVar(&dbNameOverride, "dbname", "", "dbname: name to use for Spanner db")
+	flag.StringVar(&instanceOverride, "instance", "", "instance: Spanner instance to use")
+	flag.StringVar(&filePrefix, "prefix", "", "prefix: file prefix for generated files")
+	flag.BoolVar(&verbose, "v", false, "verbose: print additional output")
+}
+
+func usage() {
+	fmt.Fprintf(os.Stderr, "Usage of %s:\n", os.Args[0])
+	flag.PrintDefaults()
+	fmt.Fprintf(os.Stderr, `Note: input is always read from stdin.
+Sample usage:
+  pg_dump mydb | %s
+  %s < my_pg_dump_file
+`, os.Args[0], os.Args[0])
+}
+
 func main() {
 	flag.Usage = usage
 	flag.Parse()
 	internal.VerboseInit(verbose)
-	now := time.Now()
 	lf, err := setupLogFile()
 	if err != nil {
 		fmt.Printf("\nCan't set up log file: %v\n", err)
 		panic(fmt.Errorf("can't set up log file"))
 	}
 	defer close(lf)
+
+	// Get project ID
```

Comment:
Do you think this is needed?
In general I thought go discouraged comments that are fairly clear from context.
If you think emphasizing the ID is important, how about renaming getProject to getProjectID?

Reply:
I agree that the comment is not necessary, but I feel that some comments can help developers to scan code quickly when the code is long. In other words, dividing the long code into different sections by using comments can improve readability. This is just my personal thought. 

For `project` or `projectID`, my original concern actually is with `instance` because we have an import alias `instance` on the top. So at the beginning, I changed all `project` and `instance` to `projectID` and `instanceID`. After that, I feel `project` and `instance` are less verbose so I changed them back. 

In addition, `getProject()` gives me a feeling that it should return a project instance. It is not very clear about what it returns from the function name.